specs-go/version.go

```diff
@@ -8,7 +8,7 @@ const (
 	// VersionMinor is for functionality in a backwards-compatible manner
 	VersionMinor = 1
 	// VersionPatch is for backwards-compatible bug fixes
-	VersionPatch = 0
+	VersionPatch = 1
 
 	// VersionDev indicates development branch. Releases will be empty string.
 	VersionDev = "+dev"
```

Comment:
Has to be empty

Reply:
Yeah, the release needs two commits -- one with `VersionDev = ""` which is tagged as the release, and then a second commit that adds `+dev` again. See 06252546d1cabcd924a5fb3cb0177178d5e3082f and 3f552ce17e77ef28a21764f7c499b555517afa43.

Reply:
Thanks for your kind explanation 🙏 I have added two commits based on your advice
- fed8226 `VersionDev = ""` and changelog
- bb43699 `VersionDev = "+dev"`